.github/workflows/label_pr.yml

```diff
@@ -20,17 +20,11 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
-      - name: Setup Python
-        uses: actions/setup-python@v4
+      - name: Setup CI env
```

Comment:
```suggestion
      - name: Setup CI environment
```

Reply:
We're using the same name everywhere so it's more consistent this way.